pegasus/sites.v3/code.org/views/professional_learning_apply_banner.haml

```diff
@@ -6,6 +6,7 @@
 - data[:nominated] = params[:nominated]
 - data[:linkSuffix] = link_suffix
 - data[:style] = style
+- data[:closingSoon] = DCDO.get("teacher_applications_closing_soon", CDO.default_teacher_applications_closing_soon)
```

Comment:
For my education, where does `data[:closingSoon]` get used?

Reply:
`ProfessionalLearningApplyBanner` component

Reply:
Ah, [sneaky](https://github.com/code-dot-org/code-dot-org/blob/44f265305b94b01351e83072629f8544e34de7c5/apps/src/sites/code.org/pages/views/professional_learning_apply_banner.js#L8).  Thanks!